.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Do we want to commit this file? I think it should be added to .gitignore

Reply:
removed